orchagent/mirrororch.cpp

```diff
@@ -336,6 +337,13 @@ bool MirrorOrch::validateSrcPortList(const string& srcPortList)
                                   p.m_alias.c_str(), port.m_alias.c_str(), srcPortList.c_str());
                         return false;
                     }
+                    portCount++;
+                }
+                if(!portCount)
```

Comment:
Please have a space after` if`. Where do we ensure that when a port is added to portchannel, the mirror session is activated?

Reply:
Pushed patchset with  space change.
session is activated updateLagMember (during SUBJECT_TYPE_LAG_MEMBER_CHANGE notification)